aggs_metric_test.go

```diff
@@ -187,5 +187,41 @@ func TestMetricAggs(t *testing.T) {
 				},
 			},
 		},
+		{
+			"top_hits agg with no source",
+			TopHits("top_hits").
+				Sorts(
+					Sorts{
+						{
+							"field_1": {
+								Order: OrderDesc,
+							},
+						},
+						{
+							"field_2": {
+								Order: OrderAsc,
+							},
+						},
+					}...,
+				).
+				SourceDisabled(true),
```

Comment:
Would you ever need to call `SourceDisabled(false)`? I'd imagine not given that you would probably call `SourceIncludes` instead. I'd be inclined to remove the boolean parameter from `SourceDisabled()` in this case.